app/search.py

```diff
@@ -6,21 +6,49 @@
 """
 
 import webbrowser
-from googlesearch import search
 import sys
+from googlesearch import search
 
 def open_browser(urls):
     for url in urls:
-        webbrowser.open_new_tab(url)
+        webbrowser.get(using='google-chrome').open(url)
```

Comment:
Always the default browser, It's okay if we can't fix the new window issue. What happens if chrome is not installed?